lib/config/index.js

```diff
@@ -8,8 +8,10 @@ logger.level = process.env.LOGGER_LEVEL || "info";
 const port = process.env.PORT || 3000;
 const githubUserAccessToken = process.env.GITHUB_USERACCESS_TOKEN;
 const eventTypes = ['IssuesEvent', 'CreateEvent', 'PullRequestEvent', 'PullRequestReviewEvent', 'PushEvent', 'WatchEvent'];
+const whitelist = ['localhost:3000', 'https://githubprofilesummary.com', 'https://www.githubprofilesummary.com'];
```

Comment:
Just tested. Change `localhost:3000` -> `http://localhost:3000`.

Reply:
You can add both, just incase.

Reply:
done